lib/rubocop/cop/style/format_string.rb

```diff
@@ -40,6 +40,55 @@ def message(detected_style)
         def method_name(style_name)
           style_name == :percent ? 'String#%' : style_name
         end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            detected_method = node.loc.selector.source.to_sym
+
+            case detected_method
+            when :%
+              autocorrect_from_percent(corrector, node)
+            when :format, :sprintf
+              case style
+              when :percent
+                autocorrect_to_percent(corrector, node)
+              when :format, :sprintf
+                corrector.replace(node.loc.selector, style.to_s)
+              end
+            end
+          end
+        end
+
+        private
+
+        def autocorrect_from_percent(corrector, node)
+          format, *_, args = *node
```

Comment:
Sorry, I'm afraid I don't understand.
Should `format` be another name, `method`?
This var is a string like "%d". 
 https://github.com/bbatsov/rubocop/pull/4354#discussion_r115272706